web/app/scripts/state/mapstate-service.js

```diff
@@ -40,7 +40,7 @@
          * @param {number} mapZoom The zoom level of the leaflet map
          */
         function setZoom(mapZoom) {
-            $log.debug('Setting zoom to: ' + JSON.stringify(mapZoom));
+            //$log.debug('Setting zoom to: ' + JSON.stringify(mapZoom));
```

Comment:
Good call noticing that these were no longer needed. Might as well just go ahead and delete them.